libraryValidations/Dotnet/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "profiles": {
+    "Dotnet": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "APP_CONFIG_VALIDATION_CONNECTION_STRING": "Endpoint=https://appconfig4ndakx7twshlm.azconfig.io;Id=A3jj;Secret=1WWpbi6jB5Bw9sYAvrHfSteqInR0UCZhwA2ynipVN1LcutGRncMyJQQJ99AJACYeBjFAArohAAACAZACIycQ"
```

Comment:
please follow https://review.learn.microsoft.com/en-us/help/platform/reference-sensitive-identifier?branch=main#replacing-non-guid-identifiers-in-your-content for dummy secrets.

If this is a real secret, rotate it right away please.

Reply:
This was accidently added in [my PR](https://github.com/microsoft/FeatureManagement/pull/16/files#diff-acb571cc85726ef428e6cd54d9cb3a0e9473337de297408b309915b101c1ce0fR6). 

It's been rotated- and the resource has been deleted anyway. 

Zhiyuan- you might need a hard refresh/force pull to clear it- since we overrode the history to rip it out as much as possible.

Reply:
Looks like a few years ago the practice was to ignore launchSettings.json. Now it's not longer ignored and secrets need to be done in the SecretManager- so this was caused by my incorrect assumption that this VS launch settings was a good place for env variables.